<launch>

  <arg name="simulated_robot" default="1"/>
  <arg name="simulated_on_gazebo" default="0"/>
  <arg name="simulated_on_rviz" default="1"/>
  <arg name="acquire_base" default="0"/>

  <param name="path" value="$(find ego_description)/urdf/model.urdf" />

  <node name="ego_model_simulated" pkg="ego_model" type="ego_model" output="screen" >
    <param name="simulated_robot" value="$(arg simulated_robot)"/>
    <param name="simulated_on_rviz" value="$(arg simulated_on_rviz)"/>
    <param name="simulated_on_gazebo" value="$(arg simulated_on_gazebo)"/>
    <param name="acquire_base" value="$(arg acquire_base)"/>
  </node> 
  

</launch>